examples/ctc-address/index.mjs

```diff
@@ -14,7 +14,7 @@ const stdlib = loadStdlib(process.env);
 
   const common = (ctcMe) => ({
     showCtcInfo: async (ctc) => {
-      const info = await ctcMe.getInfo();
+      const info = stdlib.bigNumberToNumber(await ctcMe.getInfo());
       console.log(`Contract Info From Reach      : ${ctc}`);
       console.log(`Contract Info From ctc.getInfo: ${info}`);
       stdlib.assert(ctc == info);
```

Comment:
I believe the real problem is that `==` is invalid and we should do something like turning them into JSON-ified strings and comparing those

Reply:
Or, we should add `stdlib.contract_eq`